packages/cozy-client/src/store/documents.js

```diff
@@ -221,6 +221,7 @@ export const extractAndMergeDocument = (newData, updatedStateWithIncluded) => {
   }
 
   let haveDocumentsChanged = false
+  const deeppmerge = deepmerge()
```

Comment:
I think we should create this deepmerge() object outside `extractAndMergeDocument` to avoid creating a new one for each `extractAndMergeDocument` call ?

Reply:
Yes, done in [fed634c](https://github.com/cozy/cozy-client/pull/1594/commits/fed634c0b9aefd4ba0d907edb7f0c665c0f26573)